platform/commonUI/general/src/directives/MCTPopup.js

```diff
@@ -51,6 +51,8 @@ define(
                     position = [ rect.left, rect.top ],
                     popup = popupService.display(div, position);
 
+                div.css('z-index', 75);
```

Comment:
Actually, maybe we should comment this with a //Todo mentioning that it's a workaround? It's less likely to get missed/forgotten.